internal/types/constants/trailers.go

```diff
@@ -0,0 +1,29 @@
+package constants
+
+// Keep sorted alphabetically
+const (
+	// TrailerCommitStatusActivePrefix is the prefix for trailers indicating active commit statuses.
+	TrailerCommitStatusActivePrefix = "CommitStatus-Active-"
```

Comment:
Looks like GitHub's convention is to only capitalize the first letter, e.g. `Co-authored-by`. Should we follow that? Might also mean abandoning camelcase, e.g. `Pullrequest-sourcebranch`.

Reply:
I went back and forth I kind of like the way we have it now but idk, I think Keys will probably be lower case in genreal which makes it easier to find them but we could also switch the seperator maybe?